runtests.sh

```diff
@@ -97,9 +101,11 @@ do
             echo "      --dryrun          : display the commands to the screen without running"
             echo "      --zoo             : not yet implmented"
             echo "       -j               : number of parallel jobs to run"
+            echo "       --clean          : Clean temporary files from tests"
```

Comment:
when printing this list of options `--clean` is not aligned with the others, could you change? and please add the option to `runtests.sh [--codeformat] [--black] [--black-fix] [--flake8] [--pytype] ...`